### PR TITLE
chore: improve test coverage

### PR DESCRIPTION
**Pull request by ScriptedAlchemy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,15 +11,16 @@
     "dist"
   ],
   "scripts": {
-    "prepare": "modern build",
+    "prepare": "publint && modern build",
```
Comment (2heal1) [inline package.json]: publint will check main assets if exist , can we put it after build like this: modern build && publint

File: src/browser/gradient.ts [hunk 2]
```diff
@@ -13,8 +13,7 @@ export function gradient(message: string): string[] {
   const steps = words.length - 1;
 
   if (steps === 0) {
-    console.log(`%c${message}`, `color: rgb(${startColor.join(',')}); font-weight: bold;`);
-    return [message];
+    return [`%c${message}`, `color: rgb(${startColor.join(',')}); font-weight: bold;`];
```
Comment (2heal1) [inline src/browser/gradient.ts]: it needs to console , this function will be called [here](https://github.com/2heal1/isomorphic-rslog/blob/main/src/browser/createLogger.ts#L15)

File: src/browser/createLogger.ts [hunk 3]
```diff
@@ -4,8 +4,20 @@ import { gradient } from './gradient';
...
+   }
+   if (options.level && typeof options.level !== 'string') {
+      throw new Error('Level must be a string');
+   }
+   return validatedOptions;
+}
+
 export function createLogger(options: Options = {},) {
-   return rawCreateLogger(options, {
+   const validatedOptions = validateOptions(options);
```
Comment (2heal1) [inline src/browser/createLogger.ts]: how about move the validate to rawCreateLogger ?

File: package.json (hunk 1 above)
Comment (ScriptedAlchemy) [reply package.json]: sure!

File: src/browser/gradient.ts (hunk 2 above)
Comment (ScriptedAlchemy) [reply src/browser/gradient.ts]: fixed

File: src/browser/createLogger.ts (hunk 3 above)
Comment (ScriptedAlchemy) [reply src/browser/createLogger.ts]: done

**Review by 2heal1 — APPROVED** [review]

**Pull request merged**